growthbook/growthbook.py

```diff
@@ -74,16 +91,14 @@ def set(self, key: str, value: Dict, ttl: int) -> None:
     def clear(self) -> None:
         pass
 
+class AbstractPersistentFeatureCache(AbstractFeatureCache):
+    @abstractmethod
+    def load(self) -> None:
```

Comment:
looks like implementation detail which could be hidden and called from constructor or other methods